src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -318,6 +324,79 @@ def fuzz(self,
     return LibFuzzerCommon.fuzz(self, corpus_directories, fuzz_timeout,
                                 artifact_prefix, additional_args)
 
+class FuchsiaQemuLibFuzzerRunner(new_process.ProcessRunner,LibFuzzerCommon):
+  """libFuzzer runner (when Fuchsia is the target platform)."""
+  def __init__(self, executable_path, default_args=None):
+    # This code assumes the following layout for files in the storage bucket:
+    # * multiboot.bin
+    # * fvm.blk
+    # * fuchsia-ssh.zbi
+    # * qemu/*
+    # * .ssh/*
+    local_path = os.getcwd() + "/"
+    resources_path = local_path + "fuchsia_on_clusterfuzz_resources_v1"
```

Comment:
Replace these two lines with ` resources_path = os.path.join(os.getcwd(), "fuchsia_on_clusterfuzz_resources_v1"`. `os.path.join` is a cross platform way to join directories to files/sub directories.